softdev.bib

```diff
@@ -2757,3 +2757,24 @@ @misc{watson21assessing
     month = Sep,
     url = "https://www.capabilitieslimited.co.uk/pdfs/20210917-capltd-cheri-desktop-report-version1-FINAL.pdf"
 }
+
+@article{degenbaev18cross,
+    title = "Cross-component garbage collection",
+    author = "Degenbaev, Ulan and Eisinger, Jochen and Hara, Kentaro and Hlopko, Marcel and Lippautz, Michael and Payer, Hannes",
+    year = 2018,
+    month = Oct,
+    booktitle = "OOPSLA",
```

Comment:
There should be `pages` for this I think?

Reply:
Added here: 69bab97d48c2da68f3012fb4e857490431cea217